packages/noya-state/src/reducers/themeReducer.ts

```diff
@@ -29,6 +29,11 @@ export type ThemeAction =
       id: string | string[] | undefined,
       selectionType?: SelectionType,
     ]
+  | [
+      type: `selectSymbol`,
```

Comment:
Above we have the type `ComponentsElements`. Can we add `Symbol` to that? I think symbols should support all the same actions as the other theme components

But we could also do that in a separate PR if you prefer

Reply:
I will add it, I just need to add the other actions :D

Reply:
Oh but since addSymbol it's being handled somewhere else I don't think I can add it here anymore 🤔